pkg/virt-controller/services/rendercontainer_test.go

```diff
@@ -243,6 +245,36 @@ var _ = Describe("Container spec renderer", func() {
 				Expect(specRenderer.Render(exampleCommand).LivenessProbe).To(Equal(probeWithDelay(probe)))
 			})
 		})
+
+		Context("liveness exec probe", func() {
+			It("its pod should feature the same exec probe but wrapped in a virt-probe command", func() {
```

Comment:
Could you please rephrase this sentence? Perhaps to something like "should wrap the liveness exec probe command inside virt-probe while preserving the original command"?